### DISCUSSION TITLE
can't use fonts

### DISCUSSION THREAD

**Discussion by huanmoon** [ocornut/imgui#6878] [New Users Build/Link/Run issues ONLY!]
I'm trying to use a font from ttf file.
<code>ImGuiIO& io = ImGui::GetIO();
io.Fonts -> AddFontFromFileTTF("misc/fonts/DroidSans.ttf", 1.0f, NULL, io.Fonts->GetGlyphRangesChineseFull());
</code>
I used AddFontFromFileTTF, but it doesn't work at all.
I'm sure that I didn't type the wrong name of the file

**Comment by ocornut** [comment]
You are not being specific with your question.

Have you carefully checked the Troubleshooting guide?
https://github.com/ocornut/imgui/blob/master/docs/FONTS.md#troubleshooting
